client/iotcore/mocks/Client.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2022 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+// Code generated by mockery v2.9.4. DO NOT EDIT.
```

Comment:
I remember when I got instructed that we have to use one mockery version, therefore I kindly ask: do we use 2.9.4 now?

Reply:
See utils/mockgen.sh in the repo, that's where we define the version we want to use.